py-scripts/lf_interop_throughput.py

```diff
@@ -602,81 +620,84 @@ def stop(self):
 
         self.cx_profile.stop_cx()
         self.station_profile.admin_down()
-    
+
     def pre_cleanup(self):
         self.cx_profile.cleanup()
 
     def cleanup(self):
         logger.info("cleanup done")
         self.cx_profile.cleanup()
 
-    def monitor(self,iteration,individual_df,device_names,incremental_capacity_list,overall_start_time,overall_end_time):
-        
+    def monitor(self, iteration, individual_df, device_names, incremental_capacity_list, overall_start_time, overall_end_time):
 
-        throughput, upload,download,upload_throughput,download_throughput,connections_upload, connections_download = {}, [], [],[],[],{},{}
-        drop_a, drop_a_per, drop_b, drop_b_per,state,state_of_device= [], [], [], [], [], []
-        test_stopped_by_user=False
+        throughput, upload, download, upload_throughput, download_throughput, connections_upload, connections_download = {}, [], [], [], [], {}, {}
```

Comment:
As with the other test, this is bad style and indicative that we need something like a test class object to store state. However, that's an issue for the future.